decisioncenter/dynamicdomain/README-KUBERNETES.md

```diff
@@ -3,8 +3,22 @@
 This README explains how to run the BOM dynamic domain sample in Kubernetes.
 
 #  Configuring the sample in Kubernetes
+## 1. Build the sample code 
 
-## 1. Uploading JARs on a file server
+The instructions below enable to build the JAR using a Docker container featuring Maven and a JDK version 17. 
+
+Run  command below in the `decisioncenter/dynamicdomain/src/ilog.rules.studio.samples.bomdomainpopulate` directory:
+
+This will compile the source code 
+        ```bash
+        docker run --rm \
+              -v "$(pwd)":/usr/src/sample \
+              -w /usr/src/sample \
+              maven:3.8.5-openjdk-17 \
+              mvn clean install -Dtarget.docker
```

Comment:
` -Dtarget.docker` should be removed

Reply:
my bad, i have remove it and reformat the text.